packages/tulips.yml

```
@@ -0,0 +1,6 @@
+repo: https://bitbucket.org/elaplace/tulips/src/master/
+section: domain specific libraries
+description: Visualization of the physics of stars
+site: https://astro-tulips.readthedocs.io/en/latest/# optional, default repo site
+keywords: astronomy, visualization  # optional,
+pypi_name: astro-tulips # optional, default repo name
```

Don't need to keep the comments at all.
```suggestion
site: https://astro-tulips.readthedocs.io/en/latest/
keywords: astronomy, visualization
pypi_name: astro-tulips
```